apps/spruce/src/pages/image/index.tsx

```diff
@@ -12,34 +17,45 @@ const Image: React.FC = () => {
     [slugs.tab]: ImageTabRoutes;
   }>();
 
-  if (!Object.values(ImageTabRoutes).includes(currentTab)) {
+  if (!Object.values(ImageTabRoutes).includes(currentTab) && imageId) {
     return (
       <Navigate
         replace
-        // @ts-expect-error: TODO fix in DEVPROD-7654
         to={getImageRoute(imageId, ImageTabRoutes.BuildInformation)}
       />
     );
   }
```

Comment:
I don't think this navigation is working as intended anymore; it doesn't redirect me to the BuildInformation page if the tab is empty when I load the page